tests/fixtures.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+from kagglehub.config import set_kaggle_credentials
+
+class BaseTest(unittest.TestCase):
```

Comment:
Thanks for adding a fixture for this. I like it. Should we call this `BaseTestCase` to follow the same naming pattern than the `unittest.TestCase`.